For some reason nix's pygtk doesn't have a pygtk module so we remove the version check.
If this ever changes we could remove this patch.

--- wicd-1.7.0/gtk/wicd-client.py	2009-09-01 11:05:31.000000000 -0400
+++ wicd-1.7.0/gtk/wicd-client.py	2009-12-28 00:22:57.000000000 -0500
@@ -43,9 +43,6 @@
 import atexit
 from dbus import DBusException
 
-import pygtk
-pygtk.require('2.0')
-
 HAS_NOTIFY = True
 try:
     import pynotify
